lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -89,21 +90,17 @@ class _ZoomableImageState extends State<ZoomableImage>
     Widget content;
 
     if (_imageProvider != null) {
-      content = PhotoViewGestureDetectorScope(
-        axis: Axis.vertical,
-        child: PhotoView(
-          imageProvider: _imageProvider,
-          controller: _photoViewController,
-          scaleStateChangedCallback: _scaleStateChangedCallback,
-          minScale: widget.shouldCover
-              ? PhotoViewComputedScale.covered
-              : PhotoViewComputedScale.contained,
-          gaplessPlayback: true,
-          heroAttributes: PhotoViewHeroAttributes(
-            tag: widget.tagPrefix! + _photo.tag,
-          ),
-          backgroundDecoration: widget.backgroundDecoration as BoxDecoration?,
-        ),
+      content = PhotoViewGallery.builder(
+        builder: (context, index) {
```

Comment:
All of the missing parameters (axis, controller, scaleStateChangedCallback, ...) are necessary.

Reply:
Actually, it is needed for photoView widget, for PhotoViewGallery.builder, we don't need all that. I have tested by adding that and removing it. It is of no difference @vishnukvmd

Reply:
All of these parameters are either used upstream, when they're injected from a widget above in the render tree (for eg. `minScale` and `backgroundDecoration`), or used within this same widget (for eg. `gaplessPlayback` and `controller` ensures that the thumbnail is replaced by the full-res image seamlessly).

Looking into their usages will give you a better idea.

Reply:
ok